packages/api/src/plugin.ts

```diff
@@ -81,9 +81,9 @@ export interface Plugin {
   /**
    * For grant and check permission in remote collaboration
    */
-  grantPermission?: (ctx: PluginContext) => Promise<Result<any, FxError>>;
+  grantPermission?: (ctx: PluginContext, userInfo: any) => Promise<Result<any, FxError>>;
```

Comment:
Is it possible to change this any type to a more concrete type? If you don't want to expose too much in the interface, Json type can be used instead of any.

Reply:
OK, thanks, will update